tests/testthat/test-DiseasySeason.R

```diff
@@ -196,13 +196,12 @@ test_that("$use_covid_season_v2() works", {
                            observables = observables)
 
     # Default scale
-    models <- s$get_covid_season_v1()
```

Comment:
So now only v2 is tested...

Reply:
It used to be that v1 was tested. I fixed the type in the test to now (correctly) test v2 instead
"models <- s$get_covid_season_v1()" -> "models <- s$get_covid_season_v2()"